.gitignore

```
@@ -9,3 +9,4 @@ packages/core/demo/bundle/
 packages/react/build/
 storybook-dist
 package-lock.json
+.cache/
```

Missing newline at the end of file.
```suggestion
.cache/

```